remote/github/github.go

```diff
@@ -249,6 +256,21 @@ func (g *Github) Status(u *model.User, r *model.Repo, b *model.Build, link strin
 	return err
 }
 
+func deploymentStatus(client *github.Client, r *model.Repo, b *model.Build, link string) error {
+	// the deployment ID is only available in the the link to the build as the last element in the URL
+	parts := strings.Split(b.Link,"/")
+	id, _ := strconv.Atoi(parts[len(parts)-1])
+	status := getStatus(b.Status)
+	desc := getDesc(b.Status)
+	data := github.DeploymentStatusRequest{
+		State:			github.String(status),
+		Description: 	github.String(desc),
+		TargetURL:		github.String(link),
```

Comment:
Gah, sorry. Tabs/spaces got messed up there. I'm still new to the go ecosystem - forgot to run that.